pkg/util/docker/util_docker.go

```diff
@@ -0,0 +1,27 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2017 Datadog, Inc.
+
+// +build docker
+
+package docker
+
+import (
+	"fmt"
+
+	"github.com/docker/docker/api/types/filters"
+)
+
+// buildDockerFilter creates a filter.Args object from an even
+// number of strings, used as key, value pairs
+func buildDockerFilter(args ...string) (filters.Args, error) {
+	filter := filters.NewArgs()
+	if len(args)%2 != 0 {
```

Comment:
this will work without any arguments, is the the wanted behavior?

Reply:
indeed, as we might want to create "catch-all" empty filters, will add a test for it